power_grid_model_c/power_grid_model/include/power_grid_model/main_model.hpp

```diff
@@ -49,8 +49,8 @@ class MainModel {
     };
     ~MainModel() { impl_.reset(); }
 
-    static bool is_update_independent(ConstDataset const& update_data) {
-        return Impl::is_update_independent(update_data);
+    std::vector<std::pair<std::string, bool>> is_update_independent(ConstDataset const& update_data) {
+        return impl().is_update_independent(update_data);
```

Comment:
this entire function can be deleted cause it's now unused

Reply:
Addressed.